scripts/install.cmake.sh

```diff
@@ -10,6 +10,17 @@ set -eu
 if ! [ -x /usr/bin/cmake ] || [ "${1:-}" == '--force-upgrade' ]; then
     wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | sudo tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null
     set -x
+
+    sudo tee /etc/apt/preferences.d/avoid-broken-cmake-repo-package-versions <<'APT_PREFERENCES'
+Package: cmake
+Pin: version 3.19.0-0kitware1ubuntu20.04.1
```

Comment:
This version doesn't actually exist in the upstream repos yet.
When it does, this problem will go away.
In the meantime however, we still want to install the previous version of cmake (3.18.4...).  For that to work we need to rejectlist the higher version of `cmake-data`.
With that in place, when the version above of `cmake` appears it will depend on the corresponding version of `cmake-data`, which we would have rejectlisted, causing a new but related error.
Thus, for now, we simply rejectlist both of them.